src/logistration/Logistration.jsx

```diff
@@ -111,7 +116,7 @@ const Logistration = (props) => {
                 {!institutionLogin && (
                   <h3 className="mb-4.5">{formatMessage(messages['logistration.sign.in'])}</h3>
                 )}
-                <LoginPage institutionLogin={institutionLogin} handleInstitutionLogin={handleInstitutionLogin} />
+                <LoginPageSlot institutionLogin={institutionLogin} handleInstitutionLogin={handleInstitutionLogin} />
```

Comment:
I'm using the LoginPageSlot in two places but perhaps these should be two slots that both render the LoginPage by default?